src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,13 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+
+export const TodoInfo = ({ todo }: { todo: Todo }) => (
+  <article
+    className={todo.completed
+      ? 'TodoInfo TodoInfo--completed' : 'TodoInfo'}
+  >
+    <h2 className="TodoInfo__title">{todo.title}</h2>
+
+    <UserInfo user={todo.user} />
```

Comment:
you should check if user exists before using UserInfo component 
```suggestion
    {user && <UserInfo user={todo.user} />}
```